packages/docs/src/components/unobtrusive-consent-control-ui.js

```diff
@@ -0,0 +1,197 @@
+import React, { useCallback, useEffect, useMemo, useState } from 'react'
+import { Form, Field } from 'react-final-form'
+import clsx from 'clsx'
+import { useConsentFormVisible } from '@techboi/consent-manager'
+import { FiChevronUp } from 'react-icons/fi'
+
+import { SwitchAdapter } from './form/switch'
+import styles from './unobtrusive-consent-control-ui.module.css'
+
+export function UnobtrusiveConsentControlUI({
```

Comment:
Time for some [Docusaurus TypeScript](https://v2.docusaurus.io/docs/typescript-support)? 😅

Reply:
moved to own package now, with ts :)